src/Microsoft.ML.SearchSpace/Microsoft.ML.SearchSpace.csproj

```diff
@@ -11,9 +11,4 @@
   <ItemGroup>
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonVersion)" />
   </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
-  </ItemGroup>
```

Comment:
Was this existed before only because of the `Contracts.Check`?

Reply:
Yup